airbyte-local.sh

```diff
@@ -227,33 +278,87 @@ set -eo pipefail
 main() {
     setDefaults
     parseFlags "$@"
-    if [ "$no_src_pull" = true ]; then
-        echo "Skipping pull of source image $src_docker_image"
+    if [[ "$no_src_pull" = true ]]; then
+        log "Skipping pull of source image $src_docker_image"
     else
-        echo "Pulling source image $src_docker_image"
+        log "Pulling source image $src_docker_image"
         docker pull $src_docker_image
     fi
     writeSrcConfig
     writeSrcCatalog
     parseStreamPrefix
-    if [ "$no_dst_pull" = true ]; then
-        echo "Skipping pull of destination image $dst_docker_image"
+    if [[ "$no_dst_pull" = true ]]; then
+        log "Skipping pull of destination image $dst_docker_image"
     else
-        echo "Pulling destination image $dst_docker_image"
+        log "Pulling destination image $dst_docker_image"
         docker pull $dst_docker_image
     fi
     writeDstConfig
     writeDstCatalog
     checkSrc
     loadState
 
-    if [ "$run_src_only" = true ]; then
-        echo -e "\nOnly running source"
+    if [[ "$run_src_only" = true ]]; then
+        log "Only running source"
         readSrc
     else
-        echo -e "\nRunning source and passing output to destination"
+        log "Running source and passing output to destination"
         sync
     fi
 }
 
+function fmtLog(){
+    if ((no_format)); then
+        fmtLog=""
+    else
+        fmtTime="[$(jq -r -n 'now|strflocaltime("%Y-%m-%d %T")')]"
+        if [ "$1" == "error" ]; then
+            fmtLog="$fmtTime ${RED}ERROR${NC} "
+        elif [ "$1" == "warn" ]; then
+            fmtLog="$fmtTime ${YELLOW}WARN${NC} "
+        else
+            fmtLog="$fmtTime ${BLUE}INFO${NC} "
+        fi
+    fi
```

Comment:
I needed the `no_format` option in the event CLI for testing. Do you want to allow setting it or should we remove it?